macros/get_enabled_packages.sql

```diff
@@ -13,7 +13,8 @@
     'pinterest_ads',
     'snapchat_ads',
     'tiktok_ads',
-    'twitter_ads'] %}
+    'twitter_ads',
+    'amazon_ads'] %}
```

Comment:
The particular part of me is getting the better of me haha would you be able to include this new entry in alphabetical order.

Reply:
It's the right thing to do. Done! Also I realized I only added amazon_ads to the int models but have now added them to the end models as well!